src/dashboard/ChartDashboardObject.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace luya\admin\dashboard;
+
+use Yii;
+use yii\helpers\Html;
+use yii\helpers\ArrayHelper;
+
+/**
+ * Fast generated Dashboard Objects.
+ *
+ * The default object is the default class for all {{luya\admin\base\Module::$dashboardObjects}} items without a class defintion.
+ *
+ * @author Oom <baqianxin@163.com>
+ * @since 1.0.2
+ * @example:
```

Comment:
@ example does not exists. Just put it belo the description like

```
* The default object is the default class for all {{luya\admin\base\Module::$dashboardObjects}} items without a
* class defintion.
* 
* Example usage:
* ```php
* ....
* ```
```